packages/patternfly-4/react-core/src/components/Dropdown/DropdownToggle.tsx

```diff
@@ -0,0 +1,90 @@
+import * as React from 'react';
+import { CaretDownIcon } from '@patternfly/react-icons';
+import { Toggle } from './Toggle';
+import styles from '@patternfly/react-styles/css/components/Dropdown/dropdown';
+import { css } from '@patternfly/react-styles';
+
+export interface DropdownToggleProps extends React.HTMLProps<HTMLButtonElement> {
+  /** HTML ID of dropdown toggle */
+  id?: string; 
+  /** Anything which can be rendered as dropdown toggle button */
+  children?: React.ReactNode; 
+  /** Classes applied to root element of dropdown toggle button */
+  className?: string; 
+  /** Flag to indicate if menu is opened */
+  isOpen?: boolean; 
+  /** Callback called when toggle is clicked */
+  onToggle?: (isOpen: boolean) => void; 
+  /** Element which wraps toggle */
+  parentRef?: HTMLElement; 
```

Comment:
For TabContent, we're using this as the ref type. I recall having issues when it was HTMLElement

  /** Callback for the section ref */
  innerRef?: React.Ref<any>;

Reply:
I think it works as an HTMLElement for now, can you test it? Currently Pagination's OptionsToggle uses `parentRef?: HTMLElement;` which then gets passed like: `<DropdownToggle parentRef={parentRef}>`.

Reply:
Cost Management does not use the `parentRef` property; however, I can test that the pagination component continues to work. That is, providing I can get the UI to build with these changes.

Unfortunately, I'm encountering different type errors with the build. We're using the KebabToggle here:

<img width="772" alt="Screen Shot 2019-07-19 at 10 08 10 PM" src="https://user-images.githubusercontent.com/17481322/61572864-650f3580-aa72-11e9-9dff-912cadefeb6f.png">